app/Offer.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Offer extends Model
+{
+    protected $guarded = [];
```

Comment:
add the fillable attribute to make things easier when we save the model data to the database.

Reply:
As mentioned, I am using mass assignment so an empty guarded array serves the same purpose as a fillable array with all the attribute names which will result in an error in case I didn't add it.